packages/integration_test/ios/Classes/IntegrationTestIosTest.h

```diff
@@ -21,26 +19,48 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+// For every Flutter dart test, dynamically generate an Objective-C method mirroring the test results
+// so it is reported as a native XCTest run result.
+// If the Flutter dart tests have captured screenshots, add them to the XCTest bundle.
 #define INTEGRATION_TEST_IOS_RUNNER(__test_class)                                           \
-  @interface __test_class : XCTestCase<FLTIntegrationTestScreenshotDelegate>                \
+  @interface __test_class : XCTestCase                                                      \
```

Comment:
Unfortunately since this is subclassing `XCTestCase` this code can't really live in the plugin or it will cause a "Non-public API usage" submission failure.  Keep the macro and do the test invocation logic swapping there.